src/Core/Hook/HookDispatcherInterface.php

```diff
@@ -32,7 +32,7 @@
  * Interface HookDispatcherInterface defines contract for hook dispatcher
```

Comment:
this is probably the worst idea I ever had to try to use event dispatcher to manage the hooks xD